### PR TITLE
make it possibe to configer reply-to, authorname, in send

### PR DESCRIPTION
**Pull request by kwibus**
Hi, I am working on a plug-in for kanboard to extend mail functionality. 
To make kanboard usable for us ([at computing](www.atcomputing.nl)) system for remote support.  Our plug-in would work better if you could optional  configure reply-to, author name, in send function.

this patch keeps the original behaviour of send function. but if  you can supply it with extra arguments, to make it more configurable.
  
We could do without this patch. But we hope you like it, Comments our welcome.

### CODE REVIEW

File: app/Core/Mail/Client.php [hunk 1]
```diff
@@ -44,16 +44,16 @@ public function __construct(Container $container)
      * @param  string  $html
      * @return Client
      */
-    public function send($recipientEmail, $recipientName, $subject, $html)
+    public function send($recipientEmail, $recipientName, $subject, $html, $authorName=null, $authorEmail=null)
```
Comment (fguillot) [inline app/Core/Mail/Client.php]: Please, add a space around the operator `=`.

File: app/Core/Mail/Client.php [hunk 2]
```diff
@@ -44,16 +44,16 @@ public function __construct(Container $container)
...
     {
         if (! empty($recipientEmail)) {
             $this->queueManager->push(EmailJob::getInstance($this->container)->withParams(
                 $recipientEmail,
                 $recipientName,
                 $subject,
                 $html,
-                $this->getAuthorName(),
-                $this->getAuthorEmail()
+                (is_null ($authorName) ? $this->getAuthorName() : $authorName),
```
Comment (fguillot) [inline app/Core/Mail/Client.php]: The first and last parentheses are not necessary. Can you change `is_null (` to `is_null(`?

**Comment by kwibus** [comment]
Thank you for the feedback. Hope it's good now.

**Comment by fguillot** [comment]
I was about to merge but I don't see your latest changes in the pull-request?

**Comment by kwibus** [comment]
Sorry my fault, pushed the wrong branch, second try

**Pull request merged**